### ISSUE TITLE
Add label on netbox_device_interface

### ISSUE THREAD

**Issue by jcralbino** [netbox-community/ansible_modules#508]
##### ISSUE TYPE

 - Bug Report

###### Ansible:
ansible 2.10.7
###### Netbox:
2.11.2

###### Collection:
ansible-galaxy 2.10.7
netbox.netbox 3.0.0  

##### SUMMARY
Unable to use ansible to add the label to the relevant device interface.

##### STEPS TO REPRODUCE
```
 "msg": "Unsupported parameters for (netbox.netbox.netbox_device_interface) module: label found in data. Supported parameters include: description, device, enabled, form_factor, lag, mac_address, mgmt_only, mode, mtu, name, tagged_vlans, tags, type, untagged_vlan"}
```

```
    - name: Create interface
      netbox.netbox.netbox_device_interface:
        netbox_token: "{{ lookup('vars', 'netbox_token_' ) }}"
        netbox_url: "{{ netbox_url }}"
        validate_certs: False
        data: 
          device: "{{ hostname }}" 
          name: "{{ item.name }}" 
          type: "{{ item.type }}"
          mgmt_only: false
          mode: "{{ item.mode }}"
          mac_address: "{{ item.mac_address }}" 
          label: "{{ item.label }}" 
      loop: "{{ INTERFACES }}" 
      loop_control:
        label: "{{ item.name }} type {{ item.type }} - mode {{ item.mode }}"
      tags: [update-all, update-device-lag] 
```

##### EXPECTED RESULTS

Label was introduced in the device
##### ACTUAL RESULTS

Label information not introduced

```

```

**Comment by FragmentedPacket** [comment]
Fixed in #479 and will be incorporated into 3.1.0 release.

**Comment by jcralbino** [comment]
I have tested now, and it works. Thanks
